1-Authentication/1-sign-in/README.md

```diff
@@ -0,0 +1,488 @@
+ 1. [Community Help and Support](#community-help-and-support)
+ 1. [Contributing](#contributing)
+
+## Overview
+
+This sample demonstrates a Node.js & Express web application that authenticates users against Azure AD, with the help of [Microsoft Authentication Library for Node.js](https://aka.ms/msalnode) (MSAL Node). In doing so, it illustrates authentication concepts such as [OpenID scopes](https://docs.microsoft.com/azure/active-directory/develop/v2-permissions-and-consent#openid-connect-scopes), [ID Tokens](https://docs.microsoft.com/azure/active-directory/develop/id-tokens), [ID Token validation](https://docs.microsoft.com/azure/active-directory/develop/id-tokens#validating-an-id_token) and more.
+
+## Scenario
+
+1. The client application uses **MSAL Node** to obtain an ID Token from **Azure AD**.
+2. The **ID Token** proves that the user has successfully authenticated against **Azure AD**.
+
+![Overview](./ReadmeFiles/topology.png)
+
+## Contents
+
+| File/folder           | Description                                                   |
+|-----------------------|---------------------------------------------------------------|
+| `AppCreationScripts/` | Contains Powershell scripts to automate app registration.     |
+| `ReadmeFiles/`        | Contains illustrations and screenshots.                       |
+| `App/appSettings.json`| Authentication parameters and settings                        |
+| `App/cache.json`      | Stores MSAL Node token cache data.                            |
+| `App/app.js`          | Application entry point.                                      |
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/download/) must be installed to run this sample.
+- [Visual Studio Code](https://code.visualstudio.com/download) is recommended for running and editing this sample.
+- A modern web browser. This sample uses **ES6** conventions and will not run on **Internet Explorer**.
+- An **Azure AD** tenant. For more information, see: [How to get an Azure AD tenant](https://docs.microsoft.com/azure/active-directory/develop/quickstart-create-new-tenant)
+- A user account in your **Azure AD** tenant. This sample will not work with a **personal Microsoft account**.  If you're signed in to the [Azure portal](https://portal.azure.com) with a personal Microsoft account and have not created a user account in your directory before, you will need to create one before proceeding.
+
+## Setup
+
+### Step 1: Clone or download this repository
+
+From your shell or command line:
+
+```console
+    git clone https://github.com/Azure-Samples/ms-identity-javascript-nodejs-tutorial.git
+```
+
+or download and extract the repository .zip file.
+
+> :warning: To avoid path length limitations on Windows, we recommend cloning into a directory near the root of your drive.
+
+### Step 2: Install project dependencies
+
+Locate the root of the sample folder. Then:
+
+```console
+    cd 1-Authentication\1-sign-in
+    npm install
```

Comment:
>npm install [](http://example.com/codeflow?start=4&length=11)

note that msal-express module does not install correctly after this

Reply:
problem resolved. forgot to check-in the `dist` folder previously